src/api/invitations.md

```diff
@@ -54,18 +59,22 @@ An Invitation can be used to allow users to claim ownership of a resource on the
   "resourceType": "integration",
   "exipresAt": "2021-08-26T00:02:49.488Z",
   "createdAt": "2021-08-25T00:02:49.488Z",
-  "createdBy": "crn::user:1234"
+  "createdBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey",
+  "updatedAt": "2021-08-25T00:02:49.488Z",
+  "updatedBy": "crn:WIj211vFs9cNACwBb04vQw:api-key:MyApiKey"
 }
 {% endjson %}
 
 ### Accept an Invitation **EXPERIMENTAL**
+The Account id of the user that is accepting the Invitation must be supplied.
```

Comment:
It seems like we should have an explanation here as accountId is not on the model.

Reply:
We should be adding fields as a table with the name, type and description. It's not the same as the model of the resource. See https://docs.centrapay.com/api/accounts#create-an-account for an example of this. can we do this here?

Reply:
@IsaacMurtagh have updated it, thanks